webhook/data/webhook_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <!--webhook github data-->
+        <record model="webhook" id="webhook_github">
+            <field name="name">github</field>
+            <field name="python_code_get_event">request.httprequest.headers.get('X-Github-Event')</field>
+        </record>
+        <record model="webhook.address" id="webhook_address_github">
+            <field name="name">192.30.252.0/22</field>
+            <field name="webhook_id" ref="webhook_github"/>
+        </record>
```

Comment:
This GitHub configuration should be part of the demo data, or at least with a ``noupdate="1"``?
We are testing this module, this piece of code is OK excepted the IP address (`192.30.253.0/22` here).